Description: Remove COPYING abundand licence
    This patch removes additional licence found in COPYING
Author: Dariusz Dwornikowski <dariusz.dwornikowski@cs.put.poznan.pl>
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -13,5 +13,5 @@
 add_subdirectory (parse-edid)
 
 INSTALL(FILES get-edid.1 DESTINATION share/man/man1)
-INSTALL(FILES AUTHORS ChangeLog COPYING README DESTINATION
+INSTALL(FILES AUTHORS ChangeLog README DESTINATION
 	share/doc/read-edid)
